src/main/java/org/prebid/server/bidder/nextmillennium/NextMillenniumBidder.java

```diff
@@ -51,13 +52,16 @@ public NextMillenniumBidder(String endpointUrl, JacksonMapper mapper) {
     }
 
     private static BidRequest updateBidRequest(BidRequest bidRequest, ExtImpNextMillennium ext) {
-        return BidRequest.builder()
-                .id(bidRequest.getId())
-                .test(bidRequest.getTest())
-                .ext(ExtRequest.of(ExtRequestPrebid.builder()
-                        .storedrequest(ExtStoredRequest.of(resolveStoredRequestId(bidRequest, ext)))
-                        .build()))
-                .build();
+        final ExtRequest extRequest = ExtRequest.of(ExtRequestPrebid.builder()
+                .storedrequest(ExtStoredRequest.of(resolveStoredRequestId(bidRequest, ext)))
+                .build());
+
+        final List<Imp> imps = bidRequest.getImp()
+                .stream()
+                .map(imp -> imp.toBuilder().ext(ObjectMapperProvider.mapper().valueToTree(extRequest)).build())
```

Comment:
`ObjectMapperProvider.mapper()` - you already have mapper wrapper as bidder field, just use it instead of calling static method

Reply:
Also, why do you do marshalling inside lambda? extract to local variable like so:
`final ObjectNode impExt = objectMapper.mapper().valueToTree(extRequest);`